### ISSUE TITLE
bot is offline

### ISSUE THREAD

**Issue by ayushch80** [LachlanDev/GitHub-Stats#1]

**Comment by see7e** [comment]
There is any news about this Issue? It's off for me too and the `/profile user` command isn´t responding (the language is in PT)

![image](https://user-images.githubusercontent.com/15942792/233831911-e1fb6110-a8f2-4334-afbd-d9ca8fd76161.png)
![image](https://user-images.githubusercontent.com/15942792/233831978-66c4c658-9eca-41b9-b095-8a35e0dab57a.png)

**Comment by LachlanDev** [comment]
Due to a recent issue at the location of my servers all services are offline I will be spending the next coming days bringing them all online.

**Comment by see7e** [comment]
> Due to a recent issue at the location of my servers all services are offline I will be spending the next coming days bringing them all online.

Okay @LachlanDev ! If there's anything we can do to help. Don't mind to ask!

**Comment by LachlanDev** [comment]
I should have the bot online in 10-15 mins just got some new equipment today :) thanks for the support!
 I shall close this issue once bot is up and running again

**Comment by LachlanDev** [comment]
Update: Gonna be a little longer than I anticipated have to reset up everthing. Sorry for the misunderstanding!

**Comment by see7e** [comment]
> Update: Gonna be a little longer than I anticipated have to reset up everthing. Sorry for the misunderstanding!

No problem!

**Comment by LFGaming** [comment]
any update on this issue? will the bot go online again?

**Comment by Derjyn** [comment]
Looks to be dead. Please remove from the Discord directory if this is no longer going to be maintained. Stale/abandoned resources are the worst kind to see at the top of a list when searching and such.

**Comment by CodeAsm** [comment]
yes this is anoying. thought I found the app someone used, but apparently it was not this dead app.

**Comment by LRGuess** [comment]
I might take this up and get it running again. Would anyone want to use it? Or is it a waste of my time?
I have servers to run it on already...
@CodeAsm @Derjyn

**Comment by LachlanDev** [comment]
Code has been update and the bot is online again. Invite can be found in the readme.md

**Comment by Derjyn** [comment]
How dependable is this now? It seems uptime/connection is pretty inconsistent historically. What would it take to get it long-term stability?

**Comment by LachlanDev** [comment]
@Derjyn I have nearly pushed out some new features I will ping you again once they are released.

The uptime will be more consistent from now on.

**Comment by LachlanDev** [comment]
@Derjyn The code has been fully updated, I have just brought the bot back online however commands might not show for up-to an hour due too slash commands registration process.

Find new commands list [here](https://github.com/LachlanDev/GitHub-Stats?tab=readme-ov-file#-commands--previews)

Invite the new bot [here](https://discord.com/oauth2/authorize?client_id=1368511664676274186&permissions=277025508352&scope=bot%20applications.commands). Would really appreciate your feedback on the changes to the bot

Thanks.
